apps/client/public/index.html

```diff
@@ -12,7 +12,7 @@
   <meta name="userPoolWebClientId" content="{{userPoolWebClientId}}" />
   <meta name="userPoolId" content="{{userPoolId}}" />
 
-  <title>IoT Application</title>
+  <title>IoT dashboard application</title>
```

Comment:
uppercase Dashboard Application?

Reply:
I think we want it sentence case since it's not really a proper name.